test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/SourceTelemetryTests.cs

```diff
@@ -405,6 +421,53 @@ public void VerifyProtocolDiagnosticTotals()
             telemetry[ProtocolDuration].Should().Be(3.0);
         }
 
+
+        [Fact]
+        public void LocalAndHttpSources_WithAllHttps_IsHttpsIsTrue()
```

Comment:
No longer using `IsHttps`, but rather than rename it now, figured I'd wait for feedback first.

Reply:
```suggestion
        public void LocalAndHttpSources_WithAllHttps_NumHTTPSFeeds_GreaterThanZero()
```

Reply:
Used a slightly different variation.